backend/src/main/java/com/google/sps/notifiers/EmailNotifier.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.notifiers;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.*;
+
+import java.io.IOException;
+
+/** An instance of the email notifier class is used to send emails to participants. */
+public class EmailNotifier extends Notifier {
+  private String toEmail;
```

Comment:
Since this is only assigned in the constructor now it can be made `final`. This allows for some nice optimization since the compiler can assume that the value will never change once assigned. Also please add javadocs

Reply:
Can you elaborate more on adding Javadoc I'm unsure as most of the documentation for the variables I'm using are within the constructor

Reply:
The use-cases are slightly difference. The constructor documents the constructor parameter `toEmail` which is distinct from the instance field `toEmail` even though the constructor parameter is directly assigned. Docs for this could be very similar to the constructor param.

Reply:
Understood